package.json

```diff
@@ -22,18 +22,18 @@
   "dependencies": {
     "async": "^0.9.0",
     "bluebird": "^3.4.6",
-    "debug": "^2.1.1",
+    "debug": "^3.1.0",
     "loopback-connector": "^4.2.2",
     "pg": "^7.0.0",
-    "strong-globalize": "^2.6.2",
+    "strong-globalize": "^3.1.0",
     "uuid": "^3.0.1"
   },
   "devDependencies": {
     "eslint": "^2.13.1",
     "eslint-config-loopback": "^4.0.0",
     "lodash": "^4.17.4",
     "loopback-datasource-juggler": "^3.0.0",
-    "mocha": "^2.1.0",
+    "mocha": "^3.0.0",
```

Comment:
Does Mocha `4.0.1` break something?

Reply:
I didn't try it for this repo, cuz it seems to break other repos.